treetime/gtr.py

```diff
@@ -951,7 +951,13 @@ def _exp_lt(self, t):
             Array of values exp(lambda(i) * t),
             where (i) - alphabet index (the eigenvalue number).
         """
-        return np.exp(self.mu * t * self.eigenvals)
+        log_val = self.mu * t * self.eigenvals
+        if any(i > 10 for i in log_val):
+            raise ValueError("Error in computing exp(Q * t): Q has positive eigenvalues or the branch length t \n"
```

Comment:
Same here: could we possibly know on which branch/node the positive eigenvalue happens? If we're moving from tips to root we could narrow down the area of the problem - that could be useful. But not sure whether the bug is at this step of the algorithm.